extensions/sql-database-projects/src/common/azureFunctionsUtils.ts

```diff
@@ -41,6 +41,36 @@ export async function getLocalSettingsJson(localSettingsPath: string): Promise<I
 	};
 }
 
+/**
+ * Adds a new setting to a project's local.settings.json file
+ * modified from setLocalAppSetting code from vscode-azurefunctions extension
+ * @param projectFolder full path to project folder
+ * @param key Key of the new setting
+ * @param value Value of the new setting
+ * @returns true if successful adding the new setting, false if unsuccessful
+ */
+export async function setLocalAppSetting(projectFolder: string, key: string, value: string): Promise<boolean> {
+	const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+	const settings: ILocalSettingsJson = await getLocalSettingsJson(localSettingsPath);
```

Comment:
If `getLocalSettingsJson()` throws, where are we catching it? Good to do error checking that `settings` isn't null/undefined/empty object to avoid undefined issues later

Reply:
wrapped it in a try catch to show if there's an error parsing